### PR TITLE
#3671 Add attribute tags

### PR DESCRIPTION
**Pull request by uilianries**
Changelog: Feature

This PR is related to #3671 

Also, I added `author` as default attribute when running `conan new`

Docs PR: https://github.com/conan-io/docs/pull/875

- [X] Refer to the issue that supports this Pull Request.
- [X] If the issue has missing info, explain the purpose/use case/pain/need that covers this Pull Request.
- [X] I've read the [Contributing guide](https://raw.githubusercontent.com/conan-io/conan/develop/.github/CONTRIBUTING.md).
- [X] I've followed the PEP8 style guides for Python code.
- [X] I've opened another PR in the Conan docs repo to the ``develop`` branch, documenting this one.

### CODE REVIEW

File: conans/client/cmd/new.py [hunk 1]
```diff
@@ -11,8 +11,10 @@ class {package_name}Conan(ConanFile):
     name = "{name}"
     version = "{version}"
     license = "<Put the package license here>"
+    author = "<Put your name here> <And your email here>"
     url = "<Package recipe repository url here, for issues about the package>"
     description = "<Description of {package_name} here>"
+    tags = ["<Put some tag here>", "<here>", "<and here>"]
```
Comment (memsharded) [inline conans/client/cmd/new.py]: A bit asymmetric compared to settings and other things. Should we use the tuple as default way? Maybe as the unique way (maybe support defining a single string too)?

Having multiple input formats introduces a bit more complexity, it has already bitten us in the past for other attributes, maybe we could be a bit more opinionated here this time @lasote?

Comment (uilianries) [reply conans/client/cmd/new.py]: I used the same approach from license attribute, but tuple could be nice.

both string, tuple, list are allowed. I could change to tuple and add more tests to exercise that situation.

Comment (uilianries) [reply conans/client/cmd/new.py]: updated to `topics: ('foo', 'bar', 'qux')`

**Comment by uilianries** [comment]
what does mean that error?

https://conan-ci.jfrog.info/blue/organizations/jenkins/ConanTestSuite/detail/PR-3702/4/pipeline

**Comment by danimtb** [comment]
Relaunched CI

**Comment by jgsogo** [comment]
This is adding `topics` and `homepage`, it should be mentioned in the changelog and maybe documented, shouldn't it?

**Comment by uilianries** [comment]
There is a PR related to topics on conan-io/docs but I don't know how to proceed about changelog

**Comment by jgsogo** [comment]
We generate the changelog for a version automatically by parsing all the PRs that are merged into that version, our script takes the ``Changelog: (Feature|Fix|Bugfix): whatever`` line in the first comment.

**Comment by uilianries** [comment]
nice :+1:

File: conans/test/command/info_test.py [hunk 2]
```diff
@@ -473,3 +473,58 @@ def wrong_path_parameter_test(self):
...
+
+"""
+
+        self.client.save({"subfolder/conanfile.py": conanfile})
+        self.client.run("export ./subfolder lasote/testing")
+
+        self.client.run("info ./subfolder")
+
+        self.assertIn("Pkg/0.1@PROJECT", self.client.user_io.out)
+        self.assertNotIn("License: MIT", self.client.user_io.out)
```
Comment (memsharded) [inline conans/test/command/info_test.py]: This is a bit weird, why it would be MIT license? Don't you want to check just ``License`` instead?

Comment (uilianries) [reply conans/test/command/info_test.py]: that's true, changing it now.

Comment (uilianries) [reply conans/test/command/info_test.py]: Done!

**Pull request merged**